app/i18n/docs/es/InfoModals/DecredConstitution.md

```diff
@@ -0,0 +1,38 @@
+Decred (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) es una criptomoneda segura, adaptable y autofinanciada con un sistema de gobierno comunitario integrado en su cadena de bloques. La misión del proyecto es desarrollar tecnología para el beneficio público, con un enfoque principal en la tecnología de criptomonedas. Esta constitución define un conjunto de principios que guían la toma de decisiones de las partes interesadas del proyecto y describe los procesos a través de los cuales se gobierna la cadena de bloques y el Tesoro. Este documento está destinado a gestionar las expectativas de los usuarios potenciales y reales de Decred, para que entiendan más claramente el contrato social entre ellos y el proyecto. Este documento no tiene prioridad sobre las reglas de consenso de la red Decred, en caso de que surja algún conflicto.
```

Comment:
I guess it's debatable what the interplay is between versions of the constitution written in different languages. Are all equally valid? Maybe there needs to be a notice on top about the fact that this is a translation of the original document? 

Maybe this is better off discussed in chat/politeia...